instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetServerAttributesGetter.java

```diff
@@ -21,8 +21,8 @@
   String transport(REQUEST request);
 
   @Nullable
-  Integer peerPort(REQUEST request);
```

Comment:
Similar to the client getter: how about deprecating and adding new default methods?